experimental/varfont_shader_puzzle/pubspec.yaml

```
@@ -1,69 +1,28 @@
 name: varfont_shader_puzzle
 description: A new Flutter project.
-
-# The following line prevents the package from being accidentally published to
-# pub.dev using `flutter pub publish`. This is preferred for private packages.
-publish_to: 'none' # Remove this line if you wish to publish to pub.dev
-
-# The following defines the version and build number for your application.
-# A version number is three numbers separated by dots, like 1.2.43
-# followed by an optional build number separated by a +.
-# Both the version and the builder number may be overridden in flutter
-# build by specifying --build-name and --build-number, respectively.
-# In Android, build-name is used as versionName while build-number used as versionCode.
-# Read more about Android versioning at https://developer.android.com/studio/publish/versioning
-# In iOS, build-name is used as CFBundleShortVersionString while build-number is used as CFBundleVersion.
-# Read more about iOS versioning at
-# https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
-# In Windows, build-name is used as the major, minor, and patch parts
-# of the product and file versions while build-number is used as the build suffix.
+publish_to: 'none'
 version: 1.0.0+1
 
 environment:
-  sdk: '>=2.18.6 <3.0.0'
-  flutter: ^3.7.0-1.4.pre
+  sdk: ^2.19.0-0
+  flutter: ^3.7.0
```

I guess beta is still behind stable:

```suggestion
  flutter: ^3.7.0-0
```

Or just remove it? It seems other samples don't include this constraint:

```suggestion
```

---

The reason that the original author included this constraint is that it relies on APIs that shipped in Flutter 3.7. And yes, beta is still behind stable. We should bump it soon